handlers_test.go

```diff
@@ -128,12 +143,62 @@ func TestMockOIDC_Token_CodeGrant(t *testing.T) {
 	assert.Equal(t, http.StatusUnauthorized, rrDup.Code)
 }
 
+func TestMockOIDC_Token_CodeGrant_CodeChallengeHash(t *testing.T) {
+	m, err := mockoidc.NewServer(nil)
+	assert.NoError(t, err)
+
+	codeChallenge, err := mockoidc.GenerateCodeChallenge(mockoidc.CodeChallengeMethodS256, "sum")
+	assert.NoError(t, err)
+	session, _ := m.SessionStore.NewSession(
+		"openid email profile", "nonce", mockoidc.DefaultUser(),
+		codeChallenge, mockoidc.CodeChallengeMethodS256)
```

Comment:
Do you have a Plain test scenario anywhere as well?

Reply:
Added right above this